frontend/src/components/worksheets/BundleDetail/BundleActions.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+
+
+class BundleActions extends React.Component<
+	{
```

Comment:
can we be more consistent with indentation?  I guess we're doing 4 everywhere?

Reply:
We'll run the code formatter at some point when we're all on the same page.